Task 6.py

```diff
@@ -0,0 +1,12 @@
+# Во входной строке записан текст. Словом считается последовательность
+# непробельных символов идущих подряд, слова разделены одним или большим
+# числом пробелов или символами конца строки. Определите, сколько различных
+# слов содержится в этом тексте.
+user_input = input().split()

```

Comment:
it has one line solution with sets and split

Reply:
rdy